powertools-sqs/src/main/java/software/amazon/lambda/powertools/sqs/PowertoolsSqs.java

```diff
@@ -74,12 +80,192 @@ private PowertoolsSqs() {
         R returnValue = messageFunction.apply(sqsMessages);
 
         if (deleteS3Payload) {
-            s3Pointers.forEach(SqsMessageAspect::deleteMessage);
+            s3Pointers.forEach(SqsLargeMessageAspect::deleteMessage);
         }
 
         return returnValue;
     }
 
+    /**
+     * Provides ability to set default {@link SqsClient} to be used by utility.
+     * If no default configuration is provided, client is instantiated via {@link SqsClient#create()}
+     *
+     * @param client {@link SqsClient} to be used by utility
+     */
+    public static void defaultSqsClient(SqsClient client) {
```

Comment:
is this the best name? maybe `overrideSqsClient`

Reply:
Or may be `customSqsClient` ?